delivery/router/router.js

```diff
@@ -13,18 +18,30 @@ router.use(express.json());
 router.get('/meteors', (req, res, next) => {
   if (Object.keys(req.query).length !== 0) {
     retrieveMeteorDataWithQueryParams(res, req.query)
+    .then((responseBody) => {
```

Comment:
Probably this logic can be moved to controller

Reply:
Cleaned router and moved out to separated controller layer.